File: python/hamming/hamming.py
def distance(strand_a, strand_b):
    diff = 0
    if len(strand_a) != len(strand_b):
        raise ValueError("Sequence length must be equal")
    sequence_a = list(strand_a)
    sequence_b = list(strand_b)
    for index, value in enumerate(sequence_a):
        if value != sequence_b[index]:
            diff += 1
    return diff

File: go/hamming/hamming.go
package hamming

import (
	"fmt"
)

func Distance(a, b string) (int, error) {

	if len(a) != len(b) {
		return 0, fmt.Errorf("sequence lenght msut be equal")
	}
	diff := 0
	for i := range a {
		if a[i] != b[i] {
			diff++
		}
	}

	return diff, nil

}
